scapy/supersocket.py

```diff
@@ -250,6 +250,10 @@ def sr1(self, *args, **kargs):
         """Send one packet and receive one answer
         """
         from scapy import sendrecv
+        # if not explicitly specified by the user,
+        # set threaded to False in sr1 to remove the overhead
+        # for a Thread creation
+        kargs["threaded"] = kargs.get("threaded", False)
```

Comment:
By the way, there's a native method do do that:
`kwargs.setdefault("threaded", False)`